### DISCUSSION TITLE
ON/OFF Toggle for Individual Symbols (E.G. "Parking")

### DISCUSSION THREAD

**Discussion by Scrapporoid** [organicmaps/organicmaps#1811] [Ideas]
Hi,

I see there's already a Q&A about [turning off all businesses](https://github.com/organicmaps/organicmaps/discussions/1676), so I suppose this is a cousin to that, but I'm not trying to make a duplicate discussion.  I suggest the ability to turn on/off particular symbols.

For example, "Parking" symbols.  At night (even with "Night Mode" on), the symbols are brighter than other more important map features (e.g. roadways and roadway names).  Combine this with the fact that some areas are extremely dense with parking symbols, and this causes clutter/contrast/legibility issues.

Being able to turn off "Parking" or other symbols that you don't need/use would increase usability, IMO.

Thanks for considering this!

Regards,
Shawn

**Comment by biodranik** [comment]
Then probably it makes sense to 1) improve the parking icon and 2) improve the style to avoid cluttering.

Can you please share some screenshots to better understand the context?

**Reply by Scrapporoid** [reply]
Gladly!  I don't have screenshots of the worst examples I've encountered (I deleted them and don't recall the exact locations at the moment), but here are variations on one of downtown Atlanta, Georgia, USA.  The yellow lollipop is at 33.768243, -84.388025.  First, the screenshot straight from the app; then parking removed; then churches/sightseeing/food/entertainment removed.  It's crudely done in MS Paint, but even with the names of the removed items still present, clutter/contrast/legibility is much improved.  Also note how bright the interstate (75, 85) and state (29, 78, 278) signs are, which could also do with being subdued when dark.

![Screenshot_20220108-135439](https://user-images.githubusercontent.com/97354126/148656729-6ce37dcd-b0e0-4fb0-918d-ceda7d7dddd0.png)
![Screenshot_20220108-135439_nopark](https://user-images.githubusercontent.com/97354126/148656732-94c39132-7d46-4c1b-8d9c-ccd0adaabd54.png)
![Screenshot_20220108-135439_nopark_nochurch_nosights](https://user-images.githubusercontent.com/97354126/148656737-63d1ff1f-f77d-439d-aa0c-5d31c888483e.png)

**Reply by biodranik** [reply]
Thanks! Icons should be updated, and styles definitely should be improved. At the moment, we have only one generic style for everyone. It either should be tuned for dense cities, or a separate "city" style should be added (like described in #1577).

There are also #1250 and #1031

**Reply by Scrapporoid** [reply]
Thank you for considering it, biodranik. :)
